checks/check_status.py

```diff
@@ -88,7 +88,10 @@ def logger_info():
     if len(root_logger.handlers) > 0:
         for handler in root_logger.handlers:
             if isinstance(handler, logging.StreamHandler):
-                loggers.append(handler.stream.name)
+                try:
+                    loggers.append(handler.stream.name)
+                except AttributeError:
+                    loggers.append("unnamed stream")
             if isinstance(handler, logging.handlers.SysLogHandler):
```

Comment:
I documented it in the message of https://github.com/DataDog/dd-agent/commit/378fdaef8c96c05a18a110b29ec5744d5fa309c2:

```
In `check_status.py`, stop assuming that all streams have a `name` attribute
(for instance `StringIO.StringIO` instances do not have one).
```

Do you think it deserves a comment directly in the source code?